cmd/installer/root.go

```diff
@@ -420,6 +420,50 @@ func UpdateInstalledPDSCFiles(pidxXML *xml.PidxXML, concurrency int, timeout int
 		}
 	}
 
+	pdscFiles, err = utils.ListDir(Installation.LocalDir, ".pdsc$")
```

Comment:
I am not familiar with ListDir, however why is there a `$` behind .pdsc?

Reply:
This is a regular expression. $ means that there should be nothing behind ".pdsc"